backend/backend/api/resources/account.py

```diff
@@ -1,9 +1,11 @@
 from django import http
+from django.shortcuts import get_object_or_404
 
 from rest_framework import serializers
-from rest_framework import generics
-from rest_framework import decorators
-from rest_framework import reverse as drf_reverse
+from rest_framework import viewsets
+from rest_framework.decorators import list_route
+from rest_framework.response import Response
```

Comment:
I'm not a big fan of importing symbols directly; here, we might mistake ``Response`` for Django's, for instance.